scripts/generate-usage-data/generate-usage-data.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime, timedelta, timezone
+from pprint import pprint
+import random
+import uuid
+import json
+
+import requests
+
+token = '<API Token>'
+EnterpriseCustomer = 'customer_id'
+StandardCustomer = 'customer_id'
+CustomerIds = [EnterpriseCustomer, StandardCustomer]
+userId = []
+
+for _ in range(58):
+    userId.append(uuid.uuid4().hex)
+
+def ingest(events):
+    remainder = events
+    with open('ingest.json', 'w') as f:
+        json.dump(events, f)
+    f.close()
+    print('events sent successfully')
```

Comment:
these events arent really sent, they are stored in a file.  do you want to send them?